models/enterprise_sonic/stp/sonic_stp.yaml

```diff
@@ -0,0 +1,327 @@
+                  - Number of bridges in an MST region that a BPDU can traverse before it is discarded
+                type: int
+              hello_time:
+                description:
+                  - Interval in seconds between periodic transmissions of configuration messages by designated ports
+                  - Range 1-10
+                type: int
+              max_age:
+                description:
+                  - Maximum age in seconds of the information transmitted by the bridge when it is the root bridge
+                  - Range 6-40
+                type: int
+              fwd_delay:
+                description:
+                  - Delay in seconds used by STP bridges to transition root and designated ports to forwarding
+                  - Range 4-30
+                type: int
+              mst_instances:
+                description:
+                  - Configuration for MST instances
+                type: list
+                elements: dict
+                suboptions:
+                  mst_id:
+                    description:
+                      - Value used to identify MST instance
+                    type: int
+                    required: True
+                  bridge_priority:
+                    description:
+                      - The manageable component of the bridge identifier
+                      - Value must be a multiple of 4096
+                    type: int
+                  vlans:
+                    description:
+                      - List of VLANs mapped to the MST instance. The value of a list item can be a single VLAN ID or a range of VLAN IDs
+                      - seperated by '-' or '..'; for example 70-100 or 70..100.
+                    type: list
+                    elements: str
+                  interfaces:
+                    description:
+                      - List of STP enabled interfaces
+                    type: list
+                    elements: dict
+                    suboptions:
+                      intf_name:
+                        description:
+                          - Reference to the STP interface
+                        type: str
+                        required: True
+                      cost:
+                        description:
+                          - The port's contribution, when it is the root port, to the root path cost for the bridge
+                        type: int
+                      port_priority:
+                        description:
+                          - The manageable component of the port identifier
+                        type: int
+          pvst:
+            description:
+              - Per VLAN STP configuration
+             type: list
+             elements: dict
+             suboptions:
+              vlan_id:
+                description:
+                  - VLAN identifier
+                type: int
+                required: True
+              hello_time:
```

Comment:
Here and elsewhere when only a restricted range of values is allowed, it would be helpful to specify the allowable range in the description.

This is already done for some instances in this model, but there are number of parameters for which it is currently not given.

Having this information available will make use of this model and the associated module and argspec more convenient for users.

Reply:
Addressed.

Reply:
Resolved.